### ISSUE TITLE
[Feature] Allow greater number of unit in combat & summon

### ISSUE THREAD

**Issue by TheMadSword** [GrognardsFromHell/TemplePlus#234]
Hi,

apparently a fight is limited to 32 (31 ?) units. I believe removing, or increasing to something much greater could bring interesting fight. i.e. a random encounter with a mini army of goblin/bugbear :D .

Also this should remove the limit of the 10 maximum summons (there are no D&D 3.5 rule regarding that afaik).

**Comment by pelador** [comment]
I disagree about this. The game is already CPU intensive enough. We shouldn't be looking to make the game more demanding on hardware. Secondly, it takes long enough as it is to defeat large hordes of enemies in this game.
